components/CurrentBuilding.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Why is this file added though?

Reply:
He added it for the use of the useState and useEffect functions